sourcegraph/sourcegraph
# insights: Don't allow users to delete LAM dashboard while in LAM

Closes https://github.com/sourcegraph/sourcegraph/issues/32796
Stacks on https://github.com/sourcegraph/sourcegraph/pull/32901

## Description

While in Limited Access Mode, users should not be able to:
- Delete the Limited Access Mode dashboard
- Remove insights from the Limited Access Mode dashboard

## Test plan

While in Limited Access Mode:
1. Try removing the LAM dashboard and verify that you get an error.
2. Try removing an insight from the LAM dashboard and verify that you get an error.

While not in Limited Access Mode:
1. Try removing an insight from the LAM dashboard and verify that you can.
2. Try removing the LAM dashboard and verify that you can.
